samples/stand-alone-samples/shadow/shadow_sync/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.1)
+project(shadow-sync CXX)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_INSTALL_PREFIX}/lib/cmake")
+
+file(GLOB SHADOW_SYNC_SRC
+       "*.cpp"
```

Comment:
opinion: we could make these files much more copy/pastable by renaming this something generic like SRC_FiLES instead of SHADOW_SYNC_SRC